python/tskit/trees.py

```diff
@@ -1632,6 +1633,14 @@ def draw_svg(
             node value.
         :param bool y_gridlines: Whether to plot horizontal lines behind the tree
             at each y tickmark.
+        :param bool all_edge_mutations: The edge on which a mutation occurs may span
+            multiple trees. If ``False`` or ``None`` (default) mutations are only drawn
+            on an edge if their site position exists within the genomic interval covered
+            by this tree. If ``True``, all mutations on each edge of the tree are drawn,
+            even if the their genomic position is to the left or right of the tree
+            itself (by default these "extra" mutations are drawn in a different colour).
+            Note that this means if multiple trees from a tree sequence are drawn,
```

Comment:
This is confusing I think - we don't have this option in the TreeSequence right, and we don't have any option to draw multiple trees at the same time in the API. I'd delete this last sentence.

Reply:
Yes, this is if you want to draw 2 trees separately. I do think it's worth saying, as it is a bit surprising if you don't think about it clearly, but it looks like it need rephrasing? Perhaps:

```
Note that this means that independent drawings of different trees
from the same tree sequence can share some of their plotted mutations.
```
?

Reply:
This is better, thanks.